core/src/main/scala/org/apache/spark/h2o/backends/external/ExternalBackendUtils.scala

```diff
@@ -21,13 +21,27 @@ import org.apache.spark.SparkEnv
 import org.apache.spark.h2o.H2OConf
 import org.apache.spark.h2o.backends.{ArgumentBuilder, SharedBackendUtils}
 import org.apache.spark.h2o.utils.NodeDesc
-import water.{ExternalFrameUtils, H2O}
+import water.{ExternalFrameUtils, H2O, Paxos}
 
 /**
   * Various helper methods used in the external backend
   */
 private[external] trait ExternalBackendUtils extends SharedBackendUtils {
 
+  protected def waitForCloudSize(x: Int, ms: Long): Int = {
```

Comment:
minor: x => expected_size, ms => timeout_ms?

Reply:
Thanks! We will fix it in a follow up PR.